packages/itwin/grouping-mapping-widget/src/widget/components/GroupPropertyAction.scss

```diff
@@ -2,31 +2,99 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-@import '~@itwin/itwinui-css/scss/variables';
+@import "~@itwin/itwinui-css/scss/variables";
 
 .gmw-group-property-action-container {
   display: flex;
   flex-direction: column;
   overflow-x: auto;
   height: 100%;
+  gap: $iui-baseline;
 
   .gmw-property-options {
     display: inherit;
     flex-direction: inherit;
     gap: $iui-baseline;
-    margin-bottom: $iui-baseline * 4;
   }
+}
 
-  .gmw-property-selection-container {
+.gmw-properties-list {
+  display: flex;
+  flex-direction: column;
+  gap: 7.5px;
+  flex: 1;
+  overflow-y: overlay;
+}
+
+.gmw-properties-list-surface {
+  display: flex;
+  flex-direction: column;
+  gap: 7.5px;
+  flex: 1;
+  overflow-y: overlay;
+  padding: $iui-baseline $iui-m;
+}
+
+.gmw-property-view-container {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  gap: $iui-baseline;
+  min-height: 0;
+  padding: $iui-baseline $iui-m;
+  min-width: 0;
+  .gmw-property-view-button {
+    display: flex;
+  }
+}
+
+.gmw-property-selection-container {
+  display: flex;
+  gap: $iui-s;
+  width: 45vw;
+  min-width: 100%;
+  height: 75vh;
+
+  .gutter {
```

Comment:
Non-gmw selectors. Where do they come from? What is the background-image below?

Reply:
All of these selectors were for the split library itself. I managed to change it.